pkg/collect/cluster_resources.go

```diff
@@ -443,23 +440,45 @@ func crs(
 		}
 		apiResourceList, _ := apiResourceListObj.(*metav1.APIResourceList)
 		groupVersion := apiResourceList.GroupVersion
-		for _, v := range apiResourceList.APIResources {
-			customResourceName := v.Name
-			if customResourceName != "" && !strings.ContainsAny(customResourceName, "/") {
-				fileName := customResourceName + "." + group + ".json"
-				customResourcesResponse, err := client.RESTClient().Get().AbsPath("/apis/" + groupVersion).Namespace("").Resource(customResourceName).DoRaw(ctx)
-				if err != nil {
-					errorList[fileName] = err.Error()
-					continue
-				}
-				_ = json.Unmarshal(customResourcesResponse, &customResourceItems)
-				if len(customResourceItems.Items) != 0 {
-					b, err := json.MarshalIndent(customResourceItems.Items, "", "  ")
-					if err != nil {
-						errorList[fileName] = err.Error()
-						continue
+
+		for _, namespace := range namespaces {
+			for _, resource := range apiResourceList.APIResources {
+				customResourceName := resource.Name
+				if resource.Namespaced {
```

Comment:
This should be change to first loop via resources and then check if the resource is namespaced.

The code as it is right is getting list of resources for non-namespaced types for all known namespaces.

Reply:
Fixed.